package.json

```diff
@@ -1,129 +1,129 @@
-        "@angular-devkit/build-angular": "~0.8.1",
-        "@angular-devkit/build-ng-packagr": "~0.8.1",
-        "@angular/cli": "^6.2.1",
-        "@angular/compiler-cli": "6.1.7",
-        "@angular/language-service": "6.1.7",
-        "@commitlint/cli": "^6.2.0",
-        "@commitlint/config-conventional": "^6.1.3",
-        "@commitlint/travis-cli": "^6.2.0",
-        "@semantic-release/changelog": "^2.1.2",
-        "@semantic-release/exec": "^2.3.0",
-        "@semantic-release/git": "^5.0.0",
-        "@types/d3": "^5.0.0",
-        "@types/glob": "^5.0.35",
-        "@types/jasmine": "~2.8.6",
-        "@types/jasminewd2": "~2.0.2",
-        "@types/node": "9.6.2",
-        "@types/popper.js": "^1.11.0",
-        "chalk": "^2.3.0",
-        "codelyzer": "~4.2.1",
-        "commitizen": "^2.10.1",
-        "coveralls": "^3.0.1",
-        "cp": "^0.2.0",
-        "cz-conventional-changelog": "^2.1.0",
-        "dgeni": "^0.4.9",
-        "dgeni-packages": "^0.26.2",
-        "fs-extra": "^6.0.1",
-        "glob": "^7.1.2",
-        "highlight.js": "^9.12.0",
-        "husky": "^0.14.3",
-        "jasmine-core": "~2.6.2",
-        "jasmine-spec-reporter": "~4.2.1",
-        "karma": "~1.7.0",
-        "karma-chrome-launcher": "~2.2.0",
-        "karma-cli": "~1.0.1",
-        "karma-coverage-istanbul-reporter": "^1.2.1",
-        "karma-jasmine": "1.1.1",
-        "karma-jasmine-html-reporter": "^0.2.2",
-        "lint-staged": "^7.1.2",
-        "ng-packagr": "~4.1.1",
-        "node-sass": "4.8.3",
-        "path": "^0.12.7",
-        "prettier": "^1.12.1",
-        "protractor": "~5.3.1",
-        "rxjs-tslint": "^0.1.4",
-        "semantic-release": "^15.9.5",
-        "ts-node": "^5.0.1",
-        "tsickle": ">=0.32.0",
-        "tslib": "^1.9.3",
-        "tslint": "^5.8.0",
-        "tslint-config-prettier": "^1.13.0",
-        "typescript": "2.9.2",
-        "typescript-formatter": "^7.0.1",
-        "uglify-js": "3.3.18"
-    },
-    "config": {
-        "commitizen": {
-            "path": "./node_modules/cz-conventional-changelog"
-        }
-    }
-}
+{
+    "name": "@healthcatalyst/cashmere-docs",
+    "version": "1.0.2",
+    "license": "Apache-2.0",
+    "description": "Cashmere documentation website",
+    "publishConfig": {
+        "registry": "https://registry.npmjs.org/"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/HealthCatalyst/Fabric.Cashmere.git"
+    },
+    "scripts": {
+        "ng": "ng",
+        "start": "ng serve",
+        "docs": "ts-node scripts/docs.ts && node scripts/generate-examples-docs.js && node scripts/generate-usage-docs.js",
+        "test": "ng test cashmere",
+        "test:unit": "ng test cashmere --watch=false --code-coverage --no-progress",
+        "lint": "npm run lint:tslint && npm run lint:prettier",
+        "lint:tslint": "tslint -p tsconfig.json",
+        "lint:prettier": "prettier --write '**/*.{ts,js,scss,css,md,json}'",

```

Comment:
The problem with this is that instead of checking to see if there are lint issues the server will re-write the files with the proper formatting, but it won't commit those changes. Basically it'll allow people to commit code that doesn't match our linter

Reply:
That change was accidental.  Good catch.

Reply:
fixed